codegen-on-oss/examples/context_server_example.py

```
@@ -0,0 +1,115 @@
+    
+    if response.status_code != 200:
+        print(f"Error listing snapshots: {response.text}")
+    else:
+        snapshots = response.json()["snapshots"]
+        print(f"Found {len(snapshots)} snapshots:")
+        for snapshot in snapshots:
+            print(f"  - {snapshot['snapshot_id']} ({snapshot['last_modified']})")
+
+    # Run an agent
+    print("\nRunning agent...")
+    response = requests.post(
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)